index.html

```diff
@@ -324,11 +351,36 @@
     .half-margin-bottom {
         margin-bottom: 50px;
     }
-    .name-text-container {
-        /*width: 700px;*/
-        /*padding: 0 100px;*/
-    }
 
+    .padding-container > img {
+        height: 100%;
+        width: 100%;
+        min-width: 150px;
+    }
+    .padding-container {
+        padding: 25px;
+    }
+    .tech-container {
+        min-width: 30%;
```

Comment:
```suggestion
    }
     @media screen and (min-height: 1300px) and (max-height: 2250px){
         .tech-container {
            min-width: 30%;
        }
     }
    .tech-container {
        min-width: 300px;
```
This should fix collapsing of the images a bit.
![image](https://user-images.githubusercontent.com/16579931/96517768-66538300-121e-11eb-8ab3-a0c3a6cf0686.png)

Now:
![image](https://user-images.githubusercontent.com/16579931/96517818-7c614380-121e-11eb-88e3-0c028c1b7d46.png)

and as an added bonus
![image](https://user-images.githubusercontent.com/16579931/96517859-8be08c80-121e-11eb-83b3-2340b27c2175.png)

(I wrote this on github, so double check this before committing.)